documentation/01-Prerequisites.md

```diff
@@ -54,7 +56,9 @@ Clone the repository:
 git clone https://github.com/microsoft/MLOS.git
 ```
 
-### Linux Docker Install
+### Option 1: Linux Docker Install
+
+#### Install Docker
 
 Docker is used for certain portions of the end-to-end examples and as a convient way to setup the build/dev/test environments.
```

Comment:
Because of that comment, Docker install may still deserve its own section, but I think this is ok for now.